run/namelist.restB

```diff
@@ -0,0 +1,363 @@
+
+ &param0
+ nx           =    128,
+ ny           =    256,
+ nz           =    256,
+ ppnode       =    36,
+ timeformat   =       2,
+ timestats    =       1,
+ terrain_flag = .false.,
+ procfiles    = .false.,
+ /
+
+ &param1
+ dx     =    40.0,
+ dy     =    20.0,
+ dz     =    10.0,
+ dtl    =   0.125,
+ timax  = 120.0,
+ run_time =  -999.9,
+ tapfrq =   900.0,
+ rstfrq = 10800.0,
```

Comment:
should this `rstfrq` be set to `60` here?

Reply:
You could set this to 60 if you want but it is really not crtical because the restart file that is written by the namelist.restB run is never used.